app/src/main/java/org/mozilla/fenix/components/Components.kt

```diff
@@ -107,7 +93,9 @@ class Components(private val context: Context) {
 
     val addonCollectionProvider by lazyMonitored {
         // Check if we have a customized (overridden) AMO collection (supported in Nightly & Beta)
-        if (FeatureFlags.customExtensionCollectionFeature && context.settings().amoCollectionOverrideConfigured()) {
+        if (FeatureFlags.customExtensionCollectionFeature && context.settings()
+                .amoCollectionOverrideConfigured()
+        ) {
```

Comment:
It looks like there's a few unrelated changes that may have been auto-generated by a formatter. We try to keep our commits as small and focused as possible, so please revert these unrelated changes.